db/db.js

```diff
@@ -11,12 +11,37 @@ var database = mongoose.connection;
 var orderSchema = new Schema({
   customerName:{
     type: String,
+    var testName=validator.isAlphanumeric(str [, locale]);
```

Comment:
this paramater str [, locale] should be like the 'v' in the phone number validation function